klippa/src/hmtx.rs

```diff
@@ -60,15 +72,25 @@ pub fn subset_hmtx_hhea(
     Ok(())
 }
 
-fn compute_new_num_h_metrics(hmtx: &Hmtx, gids: &IntSet<GlyphId>) -> usize {
-    let num_long_metrics = gids.len().min(0xFFFF) as usize;
-    let last_gid = gids.last().unwrap();
-    let last_advance = hmtx.advance(last_gid).unwrap();
+fn compute_new_num_h_metrics(
+    hmtx: &Hmtx,
+    gid_set: &IntSet<GlyphId>,
+    num_output_glyphs: usize,
+) -> usize {
+    let mut num_long_metrics = num_output_glyphs.min(0xFFFF) as u32;
+    let last_gid = num_long_metrics - 1;
```

Comment:
is it possible to get this far and have `num_output_glyphs` be zero?

Reply:
no, we make sure to at least insert .notdef glyph during planning.